tests/AppBundle/Entity/AdmissionPeriodEntityUnitTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Tests\AppBundle\Entity;
+
+
+use AppBundle\Entity\AdmissionPeriod;
+use AppBundle\Entity\InfoMeeting;
+use PHPUnit\Framework\TestCase;
+
+class AdmissionPeriodEntityUnitTest extends TestCase
+{
+    public function testShouldSendInfoMeetingNotification()
+    {
+        $admissionPeriod = new AdmissionPeriod();
+        $admissionPeriod->setInfoMeeting(new InfoMeeting());
+        $admissionPeriod->getInfoMeeting()->setDate(new \DateTime());
+        $admissionPeriod->getInfoMeeting()->setShowOnPage(true);
+        $this->assertTrue($admissionPeriod->shouldSendInfoMeetingNotifications());
+    }
+
+    public function testShouldSendInfoMeetingNotificationWithNull()
```

Comment:
Siden det gjøres assertFalse på shouldSendInfoMeetingNotifications, vil det være mer beskrivende å navngi denne testen testShould**Not**SendInfoMeetingNotificationWithNull?

Reply:
Hmm ser jo nå at navnet er på formatet test[function][condition]. Jeg ble forvirret av at navnet inneholder `Should`. Trodde automatisk at navnet var slik som på form nr.5 her: https://dzone.com/articles/7-popular-unit-test-naming.

> Should_ExpectedBehavior_When_StateUnderTest: This technique is also used by many as it makes it easy to read the tests. Following is how tests in first example would read like if named using this technique:
Should_ThrowException_When_AgeLessThan18
Should_FailToWithdrawMoney_ForInvalidAccount
Should_FailToAdmit_IfMandatoryFieldsAreMissing

Kan derfor bare se bort fra forslaget om å rename testen i tilfellet her, så kan vi heller evt. vurdere senere om det er en bestemt navngivning på tester vi har lyst til å gå for.

Reply:
testShould... blir bare test[metodenavn][condition] ja. Skal jeg se bort i fra navneforslaget for de andre kommentarene dine også altså?

Reply:
Ja, det gjelder de andre også 🙂

Reply:
Ja, det gjelder de andre også 🙂